Resource class exists in both Caliburn.Micro.Core and Caliburn.Micro.Platform

When using the new release 4.0.105.48049 in a MonoAndroid project MSBuild complains that the class named "Resource" is defined in both Caliburn.Micro.Core and Caliburn.Micro.Platform. It gives compiler error CS0433.

If inspecting the assemblies with Jet Brains dotPeek tool it does indeed show the class in both assemblies. 

The problem is that both classes are in the same namespace so the compiler does not know which one to use.

The Resource class is an auto generated class so I am assuming that some build task generated it for both Caliburn.Micro.Core and Caliburn.Micro.Platform when they where compiled for the MonoAndroid target.

I could resolve the problem by referencing the net standard version of Caliburn.Micro.Core.

I have made a small project that has the error in it: https://github.com/KasperSK/CaliburnTest

--- a/src/Caliburn.Micro.Core/Caliburn.Micro.Core.csproj
+++ b/src/Caliburn.Micro.Core/Caliburn.Micro.Core.csproj
@@ -19,6 +19,10 @@
   <ItemGroup Condition=" $(TargetFramework.StartsWith('MonoAndroid'))">
     <Reference Include="System.Runtime.Serialization" />
   </ItemGroup>
+  
+  <PropertyGroup Condition=" $(TargetFramework.StartsWith('MonoAndroid'))">
+    <AndroidResgenNamespace>Caliburn.Micro.Core</AndroidResgenNamespace>
+  </PropertyGroup>
 
   <ItemGroup Condition=" $(TargetFramework.StartsWith('Xamarin.iOS')) ">
     <Reference Include="System.Runtime.Serialization" />

